.github/workflows/pullrequest.yml

```
@@ -138,7 +138,7 @@ jobs:
         with:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review
-          fail-level: info
+          fail_level: info
```

actionlint (1.7.4)

input "fail_level" is not defined in action "reviewdog/action-shellcheck@v1". available inputs are "check_all_files_with_shebangs", "exclude", "fail_on_error", "filter_mode", "github_token", "level", "path", "pattern", "reporter", "reviewdog_flags", "shellcheck_flags"

(action)